extensions/mutiny/deployment/src/main/java/io/quarkus/mutiny/deployment/MutinyProcessor.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.mutiny.deployment;
+
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+
+public class MutinyProcessor {
+
+    @BuildStep
+    public FeatureBuildItem registerFeature() {
+        return new FeatureBuildItem("mutiny");
```

Comment:
The feature String is usually declared as a constant in `io.quarkus.deployment.builditem.FeatureBuildItem`